vault/logical_system.go

```diff
@@ -263,9 +263,13 @@ func NewSystemBackend(core *Core, config *logical.BackendConfig) logical.Backend
 			},
 
 			&framework.Path{
-				Pattern: "renew/(?P<lease_id>.+)",
+				Pattern: "renew" + framework.OptionalParamRegex("urllease_id"),
 
 				Fields: map[string]*framework.FieldSchema{
+					"urllease_id": &framework.FieldSchema{
```

Comment:
[Optional] I see the rationale behind `urllease_id` (urltoken, urlaccessor, etc), but somehow `url_lease_id` also seems like a better fit here.